package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "3.31.3",
+  "version": "3.31.4",
```

Comment:
Should be minor change since this is an enhancement

https://github.com/LaunchPadLab/opex-public/blob/master/gists/npm-package-guidelines.md#pull-requests-and-deployments

Reply:
@dpikt Cool. I added this link to the PR template as well. Let me know if there's a problem with that.

Reply:
Good idea!